devel/valgrind_filtered.out

```diff
@@ -0,0 +1,120 @@
+==8849== 232 bytes in 1 blocks are possibly lost in loss record 27,598 of 35,346
```

Comment:
This seems to be an output file. Is this here for any particular reason? Does it actually serve any purpose?

Reply:
Nope, must've been added accidentally. I'll remove it.